src/frontend-scripts/components/section-main/Players.jsx

```diff
@@ -251,17 +251,26 @@ class Players extends React.Component {
 				onClick={() => {
 					this.handlePlayerClick(i, player.userName);
 				}}
-				style={
-					player.customCardback &&
-					!isBlind &&
-					(!userInfo.userName || !(userInfo.userName && userInfo.gameSettings && userInfo.gameSettings.disablePlayerCardbacks))
-						? {
-								backgroundImage: `url(../images/custom-cardbacks/${player.userName}.${player.customCardback}?${player.customCardbackUid})`
-						  }
-						: {
-								backgroundImage: `url(../images/default_cardback.png)`
-						  }
-				}
+				style={(() => {

```

Comment:
Same as above for other styles, this really ought to just be a direct object with ternaries instead of this functional format.